bgpd/bgp_nht.c

```diff
@@ -1264,6 +1270,16 @@ void evaluate_paths(struct bgp_nexthop_cache *bnc)
 				    bgp_evpn_is_prefix_nht_supported(bgp_dest_get_prefix(dest)))
 					bgp_evpn_import_route(bgp_path,
 						afi, safi, bgp_dest_get_prefix(dest), path);
+
+				if (SAFI_UNICAST == safi &&
+				    (bgp_path->inst_type ==
```

Comment:
This check is the same as `inst_type != BGP_INSTANCE_TYPE_VIEW`, correct?

Reply:
ok